modules/collection/misc/gtk.nix

```
@@ -59,7 +59,7 @@ in {
         type = lines;
         default = "";
         description = ''
-          CSS to be written to '${config.directory}/.config/gtk-3.0/gtk.css'.
+          CSS to be written to `~/.config/gtk-3.0/gtk.css`.
```

```suggestion
          CSS to be written to {file}`~/.config/gtk-3.0/gtk.css`.
```

ndg supports nixpkgs flavors, and stylizes those MySt-kind roles.

---

Should I be able to nest those MySt roles? I tried doing `{file}`{env}`HOME`/.config/alacritty/alacritty.toml`.`, but that did not render correctly with `ndg`. It shows `{env} HOME` verbatim.

---

Ended up just doing `{file}`$HOME` until we get `$XDG_CONFIG_HOME` into Hjem, that should be fine. Let me know what you think.

---

No, nesting is not supported since we cannot parse it.